pkg/addons/default/aws_node_test.go

```diff
@@ -83,9 +83,26 @@ var _ = Describe("default addons - aws-node", func() {
 			Expect(err).ToNot(HaveOccurred())
 			Expect(awsNode.Spec.Template.Spec.Containers).To(HaveLen(1))
 			Expect(awsNode.Spec.Template.Spec.Containers[0].Image).To(
-				Equal("602401143452.dkr.ecr.us-east-1.amazonaws.com/amazon-k8s-cni:v1.6.0"),
+				Equal("602401143452.dkr.ecr.us-east-1.amazonaws.com/amazon-k8s-cni:v1.6.1"),
 			)
 		})
+
+		It("can update 1.12 sample for china region", func() {
+			rawClient.ClientSetUseUpdatedObjects = false // must be set for subsequent UpdateAWSNode
+
+			_, err := UpdateAWSNode(rawClient, "cn-northwest-1", false)
+			Expect(err).ToNot(HaveOccurred())
+
+			rawClient.ClientSetUseUpdatedObjects = true // for verification of updated objects
+
+			awsNode, err := rawClient.ClientSet().AppsV1().DaemonSets(metav1.NamespaceSystem).Get(AWSNode, metav1.GetOptions{})
+			Expect(err).ToNot(HaveOccurred())
+			Expect(awsNode.Spec.Template.Spec.Containers).To(HaveLen(1))
+			Expect(awsNode.Spec.Template.Spec.Containers[0].Image).To(
+				Equal("961992271922.dkr.ecr.cn-northwest-1.amazonaws.com.cn/amazon-k8s-cni:v1.6.1"),
```

Comment:
I noticed that there is one new yaml for China region https://github.com/aws/amazon-vpc-cni-k8s/blob/v1.6.1/config/v1.6/aws-k8s-cni-cn.yaml, which is the same as other region (except image name). So just add this test case to make sure it works for China region as well.

Reply:
forgot mentioning, there is slight different betwen china template and others. I have sent PR for the same in aws cni repo https://github.com/aws/amazon-vpc-cni-k8s/pull/932

Reply:
the above PR was approved, hopefully it will land soon :)